### PR TITLE
Feature/api v1 setup

### PR DESCRIPTION
**Pull request by kokuta**
#

### CODE REVIEW

File: app/Http/Controllers/BaseController.php [hunk 1]
```diff
@@ -239,26 +241,6 @@ protected function prepare_input($data)
...
     }
 
-    /**
-     * Normalizes numeric values to minimize problems for e.g. German users (using a comma instead a dot in float).
-     *
-     * @param $value the numeric string to normalize
-     *
-     * @author Patrick Reichel
-     */
-    protected function normalizeNumericString($value)
```
Comment (NinoRy) [inline app/Http/Controllers/BaseController.php:249]: why did you remove this function when it it still used in Code?

Comment (kokuta) [reply app/Http/Controllers/BaseController.php:249]: Removed by mistake during resolving conflicts.
